src/util/tests/envVariablesValidations.test.ts

```diff
@@ -54,3 +56,42 @@ describe('envVariablesValidations', () => {
     });
   });
 });
+
+describe('Validate .env file', () => {
+  const envVars = dotenv.config().parsed;
+
+  test('.env file should exist', () => {
+    expect(envVars).toBeTruthy();
+  });
+
+  test('REACT_APP_CHAIN_1 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_CHAIN_1).toBeTruthy();
+  });
+  test('REACT_APP_CHAIN_2 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_CHAIN_2).toBeTruthy();
+  });
+  test('REACT_APP_SUBSTRATE_PROVIDER_CHAIN_1 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_SUBSTRATE_PROVIDER_CHAIN_1).toBeTruthy();
+  });
+  test('REACT_APP_SUBSTRATE_PROVIDER_CHAIN_2 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_SUBSTRATE_PROVIDER_CHAIN_2).toBeTruthy();
+  });
+  test('REACT_APP_LANE_ID environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_LANE_ID).toBeTruthy();
+  });
+  test('REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_KEYRING_DEV_LOAD_ACCOUNTS).toBeTruthy();
+  });
+  test('REACT_APP_SS58_PREFIX_CHAIN_1 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_SS58_PREFIX_CHAIN_1).toBeTruthy();
+  });
+  test('REACT_APP_SS58_PREFIX_CHAIN_2 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_SS58_PREFIX_CHAIN_2).toBeTruthy();
+  });
+  test('REACT_APP_BRIDGE_ID_CHAIN_1 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_BRIDGE_ID_CHAIN_1).toBeTruthy();
+  });
+  test('REACT_APP_BRIDGE_ID_CHAIN_2 environmental variable should exist and have a value', () => {
+    expect(envVars?.REACT_APP_BRIDGE_ID_CHAIN_2).toBeTruthy();
+  });
+});
```

Comment:
We already maintain a list of expected variables in `envVariablesValidations` file. Why can't we simply call the function that validates env in this test?
I'm against duplicating the names.

Reply:
@tomusdrw The tests above are checking that the variables exist in the actual .env file (so if by mistake I delete 1 variable and it does NOT exist inside the .env file - test will fail)

In the `envVariablesValidations` file the process.env is created on the spot (instead of being imported from the actual .env file). 

You think that importing the .env and testing against that is over-testing?

Reply:
> In the envVariablesValidations file the process.env is created on the spot (instead of being imported from the actual .env file).

Not sure what you mean, `checkExpectedVariables` is simply going through the pre-defined list of variables and checking that they are available in `process.env`.

> You think that importing the .env and testing against that is over-testing?

Nope, I think we should import `.env` file but just use `checkExpectedVariables` function instead of manually listing all env variables once again.

From `dotenv` docs:
> config will read your .env file, parse the contents, assign it to process.env, and return an Object with a parsed key containing the loaded content or an error key if it failed.

So by calling `config()` you are already importing the variables to `process.env` the only change that is required is the assertions, basically the whole code should be just:
```suggestion
  test('expected ENV variables are set', () => {
    expect(checkExpectedVariables()).toBeTruthy();
  });
});
```

There is still opportunity of one more test, where we check that `.env` file does not have any unexpected variables, but that's not that much important and can be postponed until we don't have anything better to do :)